AdvancedDLSupport/NativeLibraryBase.cs

```diff
@@ -80,15 +76,15 @@ protected NativeLibraryBase
             _libLoader = libLoader;
             _symbolLoader = symLoader;
             Options = options;
-            _libraryHandle = LibraryLoader.LoadLibrary(path);
+            _libraryHandle = (libLoader ?? PlatformLoader).LoadLibrary(path);

```

Comment:
I think you misunderstood me. Assign _libraryLoader with this expression so that it only evaluates once. You still need to rename the field as well.